engines/python/setup/djl_python/transformers_neuronx.py

```diff
@@ -176,6 +139,8 @@ def load_model(self, model_type):
         else:
             self.model.to_neuron()
         os.chdir(path)
+        elapsed = time.time() - start
+        logging.info(f"Compilation completed with {elapsed}s")
```

Comment:
The info message is misleading here, I would simply replace it with `to_neuron completed with...`. `to_neuron` contains more than just compilation, but also weight loading and sharding.

Reply:
Thank you, addressed.